frontend/test/storybook/visual-regression/v-image-cell.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { expect, test } from "@playwright/test"
+
+import { makeGotoWithArgs } from "~~/test/storybook/utils/args"
+
+import type { AspectRatio } from "~/types/media"
+
+const imageCell = "a[itemprop='contentUrl']"
+// Necessary to make sure we can capture the focus state, which
+// exceeds the bounds of the actual component
+const screenshotEl = ".sb-main-padded"
+
+test.describe.configure({ mode: "parallel" })
+
+test.describe("VImageCell", () => {
```

Comment:
Awesome snapshots!
We will eventually need to add the "small" variant (where the license icons are below the image instead of inside it): [Figma](https://www.figma.com/file/bfe7wowCpMxsle7vHBAVMe/Image-results-grid-improvement?type=design&node-id=1008-23511&mode=dev).
Do you think we should add it in a different PR?

Reply:
Yup, I intend to do that in the next PR, when I introduce the change of the license icons appearing below the images.